app/views/home/index.html.haml

```diff
@@ -0,0 +1,4 @@
+%h1 Beta page for parliament.uk
+%p We're building a new website for Parliament. As we develop new features and services we will test and gather feedback on beta.parliament.uk.
+%p
```

Comment:
Add a hr and place links in a h4 for better clarity?

The links and h4 can also be on a single line like:
```ruby
%h4= link_to t('.people_link'), people_path
```